ehitchhiking-ui/src/styles.sass

```diff
@@ -28,4 +28,8 @@
 html, body
   font: $body-1 $app-font
 
```

Comment:
Pls create a task in the Trello for this bug

Reply:
😃